packages/config/config/devices/0x027a/zen71.json

```diff
@@ -0,0 +1,296 @@
+// Zooz ZEN71
+// Z-Wave Plus 700 Series S2 ON/OFF Switch
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN71",
+	"description": "Z-Wave Plus 700 Series S2 ON/OFF Switch",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xa001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On/Off",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "On/Off Paddle Orientation",
+			"description": "Reverse default on/off operation or set to toggle mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Upper paddle turns the light on, lower paddle turns it off (default)",
+					"value": 0
+				},
+				{
+					"label": "Upper paddle turns the light off, lower paddle turns it on",
+					"value": 1
+				},
+				{
+					"label": "Any paddle turns light on/off",
+					"value": 2
+				}
+			]
+		},
+		"2": {
+			"label": "LED Indicator On/Off",
+			"description": "Adjust LED function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED indicator is on when switch is off, otherwise off (default)",
+					"value": 0
+				},
+				{
+					"label": "LED indicator is on when switch is on, otherwise off",
+					"value": 1
+				},
+				{
+					"label": "LED indicator is always off",
+					"value": 2
+				},
+				{
+					"label": "LED indicator is always on",
+					"value": 3
+				}
+			]
+		},
+		"3": {
+			"label": "Auto Turn-Off Timer (Minutes)",
+			"description": "How long the switch remains on",
+			"unit": "minutes",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false
```

Comment:
needs to be `true`

Reply:
The [ZEN76 config](https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x027a/zen76.json#L38) has `false` here, which is why I corrected it—should I change that config as well?